src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  font-family: Roboto, Helvetica, sans-serif;
+  line-height: 18px;
+
+  &__image {
+    display: block;
+    margin: 32px auto 0;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__description {
+    width: 166px;
+    margin-top: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    color: $main-text-color;
+  }
+
+  &__subtitle {
+    margin-top: 4px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-text-color;
+  }
+
+  &__rate {
+    margin-top: 16px;
+
+    @include flexContainer;
+  }
+
+  &__reviews {
+    text-align: right;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__cost {
+    @include flexContainer;
+  }
+
+  &__price {
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-text-color;
+  }
+
+  &__value {
+    margin-top: 24px;
+    text-align: right;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__link {
+    margin: 16px 0;
+
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    align-items: center;
+
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+
+    text-decoration: none;
+    text-transform: uppercase;
+
+    color: rgba(255, 255, 255, 1);
+    background: $accent-color;
+    border-radius: 5px;
+
+    width: 166px;
```

Comment:
seems like you setting fixed width for almost all card items, it is a big mistake. please give padding for the whole card instead of it